recipes/aws-c-common/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.4.25":
```

Comment:
Is there any reason for the version 0.4.25? The latest version available is 0.4.34

Reply:
Those are pre-releases, so I just used the last *real* release.